driver/normalizer/normalizer.go

```diff
@@ -378,17 +377,5 @@ func (op singleQuote) Check(st *State, n nodes.Node) (bool, error) {
 	if !strings.HasPrefix(s, `'`) || !strings.HasSuffix(s, `'`) {
 		return false, nil
 	}
-	return op.op.Check(st, nodes.String(s))
-}
-
-func (op singleQuote) Construct(st *State, n nodes.Node) (nodes.Node, error) {
-	n, err := op.op.Construct(st, n)
-	if err != nil {
-		return nil, err
-	}
-	sn, ok := n.(nodes.String)
-	if !ok {
-		return nil, ErrUnexpectedType.New(nodes.String(""), n)
-	}
-	return sn, nil
+	return true, nil
```

Comment:
Could be
```golang
return strings.HasPrefix(s, `'`) && strings.HasSuffix(s, `'`), nil
```
(optional)

Reply:
So true! Sorry for missing that out, addressed in f20fe64519b06f4bb2d9efba44d6898c4f7d0451 \w history being re-written.